### PR TITLE
FH updates: gc configs, switch in ME_heppy, multicrab, plotting scripts

### PR DESCRIPTION
**Pull request by d-salerno**

### CODE REVIEW

**Comment by jpata** [comment]
Looks good Daniel, just one thing: do you really want to copy
~~~
MEAnalysis/gc/projectSkim_FH.sh
MEAnalysis/gc/count_FH.sh
~~~
In this case we'll need to maintain two versions of these things. I would keep the scripts as common as possible. What exactly would you need to change in those scripts?

**Comment by d-salerno** [comment]
Hmm, the only reason for those scripts is to keep a different output directory, that I don't want to have to remember to change each time I update the repo. Perhaps we could compromise and make the output directory somewhere inside the CMSSW directory. Maybe {CMSSW_DIR}/src/Output/count (and skim)?

File: MEAnalysis/python/MEAnalysis_heppy.py [hunk 1]
```diff
@@ -23,6 +23,9 @@
     print "Loading ME config from", os.environ["ME_CONF"]
     meconf = imp.load_source("meconf", os.environ["ME_CONF"])
     from meconf import Conf
+elif 0:
```
Comment (jpata) [inline MEAnalysis/python/MEAnalysis_heppy.py]: @d-salerno why is this `elif 0` here? if you want to run tests locally, you could write it in the Makefile, like
~~~
test_MEAnalysis_FH:
        ME_CONF=MEAnalysis/python/cfg_FH.py python MEAnalysis/python/MEAnalysis_heppy.py
~~~

let's avoid strange looking dead code that we always have to remember to comment in or out.

**Comment by jpata** [comment]
If you remove the copying of the files to `_FH`, I can merge this. PR #68 takes care of the output dir stuff.

File: MEAnalysis/python/MEAnalysis_heppy.py (hunk 1 above)
Comment (d-salerno) [inline MEAnalysis/python/MEAnalysis_heppy.py]: it's for crab - I would make it elif 1 when I want to run. If you know of a way to put the switch in the crab submission, that would be better.

Comment (jpata) [inline MEAnalysis/python/MEAnalysis_heppy.py]: OK, I will merge this now, but lets find a better way to configure crab with a different MEM config.

**Comment by jpata** [comment]
Regarding `MEAnalysis/gc/confs/QCD*.conf`, if they have pretty similar runtimes per file, you could as well merge the configurations and specify the datasets like this:

~~~
dataset =
    /path/to/qcd1.dat
    /path/to/qcd2.dat
~~~
see https://github.com/jpata/tthbb13/blob/master/MEAnalysis/gc/confs/projectSkim.conf#L16

**Pull request merged**